### PR TITLE
Utils

### LINKED ISSUE #2392: Utils
Migrate 'clippedObj' function into Hypersearch utils.

### PR DESCRIPTION
**Pull request by mihail911**
Fixes #2392

### CODE REVIEW

**Comment by mihail911** [comment]
@scottpurdy

File: nupic/support/__init__.py [hunk 1]
```diff
@@ -566,76 +566,37 @@ def handleErrorPatch(*args, **kwargs):
...
-    objOut = str(obj)
-    if len(objOut) > maxElementSize:
-      objOut = objOut[0:maxElementSize] + '...'
-
-  return objOut
 
 
 
-def intTo8ByteArray(inValue):
+def intto8bytearray(invalue):
```
Comment (scottpurdy) [inline nupic/support/__init__.py]: Was all of the lowercasing in this file intentional? It doesn't seem like a good change.

File: nupic/swarming/HypersearchV2.py [hunk 2]
```diff
@@ -36,13 +36,13 @@
...
-from nupic.support import clippedObj
+from nupic.swarming.hypersearch.utils import clippedObj
 from nupic.support.serializationutils import sortedJSONDumpS
 from nupic.support.configuration import Configuration
 from nupic.swarming.hypersearch.errorcodes import ErrorCodes
 from nupic.database.ClientJobsDAO import (
     ClientJobsDAO, InvalidConnectionException)
-from nupic.swarming.utils import (runModelGivenBaseAndParams,
+from nupic.swarming.hypersearch.utils import (runModelGivenBaseAndParams,
                                                   runDummyModel)
```
Comment (scottpurdy) [inline nupic/swarming/HypersearchV2.py]: line up the wrapped line correctly

**Comment by scottpurdy** [comment]
done with review

File: nupic/support/__init__.py (hunk 1 above)
Comment (mihail911) [inline nupic/support/__init__.py]: No it was not. I honestly have no clue how this happened. I will change it back to the appropriate casing.

**Comment by mihail911** [comment]
@scottpurdy

**Pull request merged**